src/utils.ts

```diff
@@ -1,7 +1,9 @@
+import { AppTab } from "./types/tab.type";
+
 const STORAGE_KEYS = {
   SEARCHED_REPO_NAME: 'gch-searched-repo',
 };
-export const getRepoNameFromUrl = (tab: chrome.tabs.Tab, repo = '') => {
+export const getRepoNameFromUrl = (tab: AppTab, repo = '') => {
   const urlChunks = tab.url!.split('files');
   let repoName = urlChunks[0];
```

Comment:
Not really related to your commit, but why isn't this a constant variable?

Reply:
I'm not clear on what you're referring to. The `AppTab` type? `repo`?  or the `tab` variable etc?

Reply:
this one ->  let repoName = urlChunks[0];